packages/utils/docs/report.md

```diff
@@ -0,0 +1,93 @@
+# Code Pushup Report
+
+<!-- START: Meta Section -->
+
+---
+
+_Version: 0.0.1_  
+_Date: 9/11/2023 12:00:00am (430ms)_  
+_Plugins: 3; Audits: 44_
+
+---
+
+<!-- STOP: Meta Section -->
+
+<!-- START: Overview Section -->
+<!-- Only audits with weight > 0 are listed in the overview section -->
+
+<!-- Overview Version 1 -->
+
+| Performance | A11y | NxAdoption |
+| ----------- | ---- | ---------- |
+| 60          | 34   | 80         |
+
+<!-- Overview Version 2 -->
+
+| Category    | Score | Audits |
+| ----------- | ----- | ------ |
+| Performance | 60    | (3/4)  |
+| A11y        | 34    | (7/24) |
+| NxAdoption  | 80    | (7/12) |
+
+<!-- Overview Version 3 -->
+
+---
+
+Performance: **60**  
+A11y: **34**  
+NxAdoption: **80**
+
+---
+
+<!-- STOP: Overview Section -->
+
+<!-- START: Details Section -->
+
+**Performance: 60** (3/4)
+
+- <details>
+  <summary>Largest Contentful Paint</summary>
+  Largest Contentful Paint (LCP) is an important, stable Core Web Vital metric for measuring [perceived load speed](https://web.dev/user-centric-performance-metrics/#types-of-metrics) because it marks the point in the page load timeline when the page's main content has likely loaded—a fast LCP helps reassure the user that the page is useful.
+  </details>
+- <details>
+  <summary>Cumulative Layout Shift</summary>
+  Cumulative Layout Shift (CLS) is a stable Core Web Vital metric. It is an important, user-centric metric for measuring visual stability because it helps quantify how often users experience unexpected layout shifts—a low CLS helps ensure that the page is delightful.
+  </details>
+- <details>
+  <summary>Interaction to Next Paint</summary>
+  </details>
+- <details>
+  <summary>Time to First Byte</summary>
+  </details>
+
+**A11y 34** (7/24)
+
+- <details>
+  <summary>Audit 1</summary>
+  </details>
+- <details>
+  <summary>Audit 2</summary>
+  </details>
+- <details>
+  <summary>Audit 3</summary>
+  </details>
+- <details>
+  <summary>Audit 4</summary>
+  </details>
+
+**NxAdoption 80** (7/12)
+
+- <details>
+  <summary>Audit 1</summary>
+  </details>
+- <details>
+  <summary>Audit 2</summary>
+  </details>
+
+<!-- STOP: Details Section -->
+
+<!-- START: Footer Section -->
+
+Code Pushup Cloud ID: [123abc456def]()
```

Comment:
I would skip this for now, it'll be some quite time before we can support such a flow, I think.